### PR TITLE
chore: remove microshift installation

### PR DESCRIPTION
**Pull request by vprashar2929**
This PR removes the configuring and setting up microshift for local development
As discussed here:
https://github.com/sustainable-computing-io/local-dev-cluster/pull/80#issuecomment-2090252643

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,23 +39,13 @@ Will setup container runtime on your host instance.
 
 1. Modify kind [config](./kind/manifests/kind.yml) to make sure `extraMounts:` cover
    the linux header and BCC.
-1. Export `CLUSTER_PROVIDER` env variable:
-
-    ```bash
```
Comment (SamYuan1990) [inline README.md:44]: todo: update this part later on.

File: README.md [hunk 2]
```diff
@@ -81,10 +69,6 @@ Will setup container runtime on your host instance.
     ./main.sh up
     ```
 
-1. kubeconfig
```
Comment (SamYuan1990) [inline README.md:84]: we should keep this.

**Review by SamYuan1990 — APPROVED** [review]
we can update readme back in another PR.

File: README.md (hunk 2 above)
Comment (vprashar2929) [reply README.md:84]: Can you explain why we need to keep this?

Comment (SamYuan1990) [reply README.md:84]: we still have kind provider in this repo, so when people use kind to set up their cluster with this repo. he/she should get kubeconfig for kind.

**Pull request merged**